daemon/k8s_watcher.go

```diff
@@ -246,6 +348,51 @@ func (d *Daemon) deleteK8sNetworkPolicy(obj interface{}) {
 	}
 }
 
+// addK8sNetworkPolicyDeprecated FIXME remove in k8s 1.8
+func (d *Daemon) addK8sNetworkPolicyDeprecated(obj interface{}) {
+	k8sNP, ok := obj.(*v1beta1.NetworkPolicy)
+	if !ok {
+		log.Errorf("Ignoring invalid k8s v1beta1 NetworkPolicy addition")
+		return
+	}
+	rules, err := k8s.ParseNetworkPolicyDeprecated(k8sNP)
+	if err != nil {
+		log.Errorf("Error while parsing kubernetes v1beta1 network policy %+v: %s", obj, err)
+		return
+	}
+
+	opts := AddOptions{Replace: true}
+	if _, err := d.PolicyAdd(rules, &opts); err != nil {
+		log.Errorf("Error while adding kubernetes v1beta1 network policy %+v: %s", rules, err)
+		return
+	}
+
+	log.Infof("Kubernetes v1beta1 network policy '%s' successfully added", k8sNP.Name)
+}
+
+// updateK8sNetworkPolicyDeprecated FIXME remove in k8s 1.8
+func (d *Daemon) updateK8sNetworkPolicyDeprecated(oldObj interface{}, newObj interface{}) {
+	log.Debugf("Modified v1beta1 policy %+v->%+v", oldObj, newObj)
+	d.addK8sNetworkPolicyDeprecated(newObj)
+}
+
+// deleteK8sNetworkPolicyDeprecated FIXME remove in k8s 1.8
+func (d *Daemon) deleteK8sNetworkPolicyDeprecated(obj interface{}) {
+	k8sNP, ok := obj.(*v1beta1.NetworkPolicy)
+	if !ok {
+		log.Errorf("Ignoring invalid k8s v1beta1.NetworkPolicy deletion")
+		return
+	}
+
+	labels := labels.ParseSelectLabelArray(k8s.ExtractPolicyNameDeprecated(k8sNP))
+
+	if _, err := d.PolicyDelete(labels); err != nil {
+		log.Errorf("Error while deleting v1beta1 kubernetes network policy %+v: %s", labels, err)
```

Comment:
General comment - a lot of these logs are inconsistent in how they phrase things, for example:
`v1beta1 kubernetes network policy` vs `kubernetes v1beta1`, `NetworkPolicy` vs `network policy`, `kubernetes` vs. `k8s`.

I think it's important for us to stick to convention when writing these logs, esp. for when we are searching logs to debug. If there is not a convention, we might miss out on key information.